src/client/lib/consentsTracking.ts

```diff
@@ -0,0 +1,79 @@
+import { PageData } from '@/shared/model/GlobalState';
+import { Routes } from '@/shared/model/Routes';
+import { sendOphanInteractionEvent } from './ophan';
+
+const trackInputElementInteraction = (
+  inputElem: HTMLInputElement,
+  component: string,
+  consentName: string,
+) => {
```

Comment:
Should the return type be `void` for all of these functions ?

Reply:
Yes, I'll add this to the type